variables.tf

```
@@ -46,6 +46,12 @@ variable "source_security_group_ids" {
   description = "Set of EC2 Security Group IDs to allow SSH access (port 22) to the worker nodes. If you specify `ec2_ssh_key`, but do not specify this configuration when you create an EKS Node Group, port 22 on the worker nodes is opened to the Internet (0.0.0.0/0)"
 }
 
+variable "additional_security_group_ids" {
```

We have [a new naming convention](https://github.com/cloudposse/terraform-aws-security-group/pull/17#issue-678450826). Please rename this `associated_security_group_ids`
```suggestion
variable "associated_security_group_ids" {
```